e2echart/e2e-chart-template.html

```diff
@@ -95,14 +95,17 @@ <h5 class="modal-title">Resource</h5>
     }
 
     function isEndpointConnectivity(eventInterval) {
+        if (eventInterval.locator.includes("disruption/")) {
```

Comment:
the standard disruption backend now uses this.  If it works, then we'll be able to remove the special casing for apiservers.